.github/actions/preload-img-cache/action.yml

```diff
@@ -1,57 +1,38 @@
 name: Preload img cache
 inputs:
+  cache-key-prefix:
+    description: 'Device cache-key prefix'
+    required: true
   device:
     description: 'Device name'
     required: true
-  filename:
-    description: 'Name of file in workdir'
-    required: true
-  test-db-path:
-    description: 'Path to test database'
-    required: true
-outputs:
-  cache-hit:
-    description: 'Did we hit the cache?'
-    value: ${{ steps.cache-img.outputs.cache-hit }}
 
 runs:
   using: "composite"
   steps:
     - uses: actions/cache@v3
       id: cache-img
       with:
-        key: img-v2-${{ hashFiles(inputs.test-db-path) }}-${{ inputs.device }}
-        path: |
-          workdir/${{ inputs.filename }}-sparse.tar
-    - if: ${{ steps.cache-img.outputs.cache-hit }}
```

Comment:
If we want to make use of the sparse images in the Github cache, this (and the similar post step) should be added back in as the actions/cache action doesn't preserve sparse files.

Before running the next CI the caches should be invalidated by adding a version key or manually delete the cache.

Reply:
Ah shoot, I got a bit too hasty when I saw actions/cache@v3 used tar internally and missed that it doesn't use the sparse flag. I'll add it back.